lib/ansible/modules/notification/mail.py

```diff
@@ -208,24 +199,25 @@
 def main():
 
     module = AnsibleModule(
-        argument_spec = dict(
-            username = dict(default=None),
-            password = dict(default=None, no_log=True),
-            host = dict(default='localhost'),
-            port = dict(default=25, type='int'),
-            sender = dict(default='root', aliases=['from']),
-            to = dict(default='root', aliases=['recipients']),
-            cc = dict(default=None),
-            bcc = dict(default=None),
-            subject = dict(required=True, aliases=['msg']),
-            body = dict(default=None),
-            attach = dict(default=None),
-            headers = dict(default=None),
-            charset = dict(default='us-ascii'),
-            subtype = dict(default='plain'),
-            secure = dict(default='try', choices=['always', 'never', 'try', 'starttls'], type='str'),
-            timeout = dict(default=20, type='int')
-        )
+        argument_spec=dict(
+            username=dict(type='str'),
+            password=dict(type='str', no_log=True),
+            host=dict(type='str', default='localhost'),
+            port=dict(type='int', default=25),
+            sender=dict(type='str', default='root', aliases=['from']),
+            to=dict(type='list', default=['root'], aliases=['recipients']),
+            cc=dict(type='list', default=[]),
+            bcc=dict(type='list', default=[]),
+            subject=dict(type='str', required=True, aliases=['msg']),
+            body=dict(type='str'),
+            attach=dict(type='list', default=[]),
+            headers=dict(type='list', default=[]),
+            charset=dict(type='str', default='us-ascii'),
```

Comment:
I believe using `utf-8` by default will be ok for this.

Reply:
I think so too.